prom2teams/app/teams_client.py

```diff
@@ -1,20 +1,55 @@
+import json
+import logging
 import requests
+from tenacity import retry, wait_fixed, after_log
 
 from .exceptions import MicrosoftTeamsRequestException
 
-session = requests.Session()
-session.headers.update({'Content-Type': 'application/json'})
-
-
-def post(teams_webhook_url, message):
-    response = session.post(teams_webhook_url, data=message, timeout=(5,20))
-    if not response.ok or response.text is not '1':
-        exception_msg = 'Error performing request to: {}.\n' \
-                        ' Returned status code: {}.\n' \
-                        ' Returned data: {}\n' \
-                        ' Sent message: {}\n'
-        raise MicrosoftTeamsRequestException(exception_msg.format(teams_webhook_url,
-                                                                  str(response.status_code),
-                                                                  str(response.text),
-                                                                  str(message)),
-                                             code=response.status_code)
+logger = logging.getLogger('prom2teams')
```

Comment:
Other files call the logger variable "log", it is better if we maintain the same naming convention.